spec/demo_data/fake_student_spec.rb

```diff
@@ -18,10 +18,23 @@
     expect(student.home_language).not_to be_nil
   end
 
+  it "sets student enrollment status to be either 'Active' or 'Transferred'" do
+    expect(student.enrollment_status).to eq('Active').or(eq('Transferred'))
+  end
+
+  it "sets student race attributes" do
+    expect(student.race).not_to be_nil
+  end
+
   it 'adds student assessments' do
     expect(student.student_assessments).not_to be_empty
   end
 
+  it 'adds student attendance events' do
+    expect(student.absences).not_to be_empty
```

Comment:
This expectation isn't accurate. Some students in the fake data have no absences or tardies, just like some students in the real production data. Check out the distribution function in the method that generates fake attendance events.